pkg/proxy/proxier.go

```diff
@@ -319,38 +319,43 @@ type Proxier struct {
 	hostIP        net.IP
 }
 
+var (
+	ErrProxyOnLocalhost = fmt.Errorf("cannot proxy on localhost")
```

Comment:
We was "the proxier".  I wanted to check on ErrProxyOnLocalhost since it's doing argument validation and we want to give a very specific message to users.  ProxyLocked I wanted to test on.  Initialize and test on were generic errors - yeah it failed, we can't start the proxy because of some arbitrary iptables craziness.

Basically the difference between http 422 and 404 - 422 means something very specific, 404 could mean anything.  The ProxyOnLocalhost was validation, the others were "we can't start because of :shrug:")